tutorial/stream-client.md

```diff
@@ -0,0 +1,276 @@
+(tut-set-up-stream-client)=
+
+# Set up a stream client
+
+This tutorial guides you through the process of setting up a web-based streaming client using the Anbox Cloud streaming stack. The connection between the stream client and the server uses WebRTC backed by WebSockets that enable the real time communications required for streaming. To know more about the WebRTC configuration, see {ref}`ref-webrtc`.
+
+For security reasons, there are limits on what APIs are exposed through the reverse proxy configured by the appliance. The API provided by the Anbox Stream Gateway is typically meant to be used in service-to-service communication and not on public endpoints. If you want to use the API from a client application, you should use a proxy service that communicates with the Anbox Stream Gateway and also provides the user with an authentication method specific to your service. We will see how to set this up in this tutorial.
+
+## Preparation
+
+Complete the following preparatory steps:
+
+### Install the Anbox Cloud Appliance
+
+We need the Anbox Cloud streaming stack to be deployed already to set up a streaming client. So let's get the streaming stack ready by installing the Anbox Cloud Appliance. Follow the instructions in the {ref}`tut-installing-appliance` until initialising the appliance. When you have installed and initialised the appliance, let's proceed to the next step.
+
+(sec-create-access-token)=
+### Create an access token
+
+To access the HTTP API of the Anbox Cloud stream gateway, an access token is required. Each access token is associated with a service account.
+
+On the machine where Anbox Cloud Appliance is installed, create the service account by running the following command:
+
+    sudo anbox-cloud-appliance gateway account create streaming-tutorial
```

Comment:
->
`sudo anbox-cloud-appliance.gateway account create streaming-tutorial`
I don't think we provide the subcommand `gateway` to the anbox-cloud-appliance snap

Reply:
Great catch, thanks!